cdflow_commands/config.py

```diff
@@ -49,7 +57,9 @@ def load_service_metadata():
                 metadata['TYPE'],
                 metadata['REGION'],
                 metadata['ACCOUNT_PREFIX'],
-                metadata.get('ECS_CLUSTER', 'default')
+                metadata.get('ECS_CLUSTER', 'default'),
+                metadata.get('HANDLER', ''),
+                metadata.get('RUNTIME', '')
```

Comment:
I think perhaps for some of the things in the `service.json` manifest we could nest them under e.g. `service_parameters` and pass the whole lot to Terraform. What do you think?